botCommands/mockData.js

```diff
@@ -0,0 +1,79 @@
+const { Collection, ClientUser } = require("discord.js")
+
+
+
+const zeroMentionData = new Collection()
+
+const singleMentionData = new Collection()
```

Comment:
Could we make this a function we can call to return the amount of mentions we want?

Reply:
Didn't think of that, will do!